### DISCUSSION TITLE
How to remove DIV with a specific class from output?

### DISCUSSION THREAD

**Discussion by sewid** [thephpleague/commonmark#636] [Q&A] [answered]
## Question

Hi there,

I'm try to figure out a solution for the following problem. This is my markdown code:

```
Some Text

More Text

<div class="hidden">

	<div class="example">

		Some more content

	</div>

</div>

Something else
```

I want to exclude everything withing the div with class "hidden" from converting to HTML. 

As result, I would expect:
```
<p>Some Text</p>
<p>More Text</p>
<p>Something else</p>
```

Can somebody help me, how that can be done? I tried Listener, Parser and Renderer - nothing works, as I want.

Thanks in advance
Sebastian

**Comment by markhalliwell** [comment, marked as answer]
While Markdown allows raw HTML, it doesn't parse it. You will need a different library (or even just use [DOMDocument](https://www.php.net/manual/en/class.domdocument.php)) to traverse the HTML and remove it that way.
